www/js/intro.js

```diff
@@ -83,10 +86,23 @@ angular.module('emission.intro', ['emission.splash.startprefs',
     window.cordova.plugins.BEMJWTAuth.signIn().then(function(userEmail) {
       // ionicToast.show(message, position, stick, time);
       // $scope.next();
-      ionicToast.show(userEmail, 'middle', false, 2500);
+      $scope.userEmail = userEmail;
       CommHelper.registerUser(function(successResult) {
-        $scope.finish();
+          const uuid = successResult.uuid;
+          return EnketoSurvey.getAllSurveyAnswers("manual/user_profile_survey"
```

Comment:
Recall that you can use CRUD operations directly for the profile instead of adding an entry into the timestream.